content/docs/introduction/design-doc.md

```diff
@@ -3,51 +3,44 @@ title: Design Documents
-| [Prometheus timezones support](https://docs.google.com/document/d/1xfw1Lb1GIRZB_-4iFVGkgwnpwuBemWfxYqFdBm7APsE/edit) | 2021‑05‑29 | Proposed | |
-| [Moving to goreleaser](https://docs.google.com/document/d/16LOT2wK-jntlU-EFADfaEF3YbKH81U9Zl_PvSu4qVwo/edit) | 2021‑06‑05 | Proposed |
-| [Alertmanager Log Receiver](https://docs.google.com/document/d/1Oevu2stHVGAupzmc9C7_wW5nTb_CJ6Ut72viXfve6zI/edit) | 2021‑06‑10 | Proposed |
-| [Extra HTTP parameters in the blackbox exporter](https://docs.google.com/document/d/1VwqXi2TOb5KXaZY6Iio7411x64pJao3GusX8MqYsJ2g/edit) | 2021‑06‑23 | Proposed | |
-| [Making durations and number literals the same](https://docs.google.com/document/d/1LaZfknXuuRWGtQSbULoMtclQhuLUMrdwg15wMvoBvCQ/edit) | 2021‑07‑26 | In progress | [#9138](https://github.com/prometheus/prometheus/pull/9138) |
-| [Metadata](https://docs.google.com/document/d/1XiZePSjwU4X5iaIgCIvLzljJzl8lRAdvputuborUcaQ/edit) | | TODO | |
-| [OpenMetrics transition](https://docs.google.com/document/d/1VuUBdRyIDR2uID2j2abWGt6PbWu7o-s06btZgrwH4vQ/edit?usp=sharing) | 2022-06-24 | In progress | |
-| Semantics of muting in Alertmanager | | TODO | |
-| [Extrapolation in range selectors (xrate)](https://docs.google.com/document/d/1y2Mp041_2v0blnKnZk7keCnJZICeK2YWUQuXH_m4DVc/edit#) | | Under review | |
-| Serverless, MQTT, and IoT use cases in the Prometheus ecosystem | | TODO | |
-| Static arithmetic for timestamps and durations | | TODO | |
-| [Prometheus long-term supported releases](https://docs.google.com/document/d/1wCXLnvh460UG10Mw019UbYMWDwSWWdvgwY61LHY0A0w/edit) | 2022-03-02 | Proposed | |
-| [Support for out of order samples in the TSDB](https://docs.google.com/document/d/1Kppm7qL9C-BJB1j6yb6-9ObG3AbdZnFUBYPNNWwDBYM/edit) | 2022-04-13 | Implemented | [#11075](https://github.com/prometheus/prometheus/pull/11075) |
-| [Structured Remote Write Protocol](https://docs.google.com/document/d/1i880vXorPec1ls91bnuoKR3-Hl3-D9XA1q1w734rZnU/edit#) | 2022-04-25 | proposed | |
+# Design Documents
+
+It's essential to clearly explain the reasons behind certain design decisions to have a community consensus. This is especially
+important in Prometheus, where every decision might have a significant impact given the high adoption and stability of the software and standards we work on.
+
+In our world, no decision is perfect, so having a design document that explains the trade-offs we made is essential.
+Such a document can also be used later on as a reference and for knowledge-sharing purposes.
+
+Note that design documents do not always reflect exactly what has been (or will be) implemented. Implementation details
+might have changed since a feature was merged. Design docs are not considered documentation and can not define a standard.
+Instead, it should explain the motivation, scope, the decision made and alternatives considered.
```

Comment:
decision → decisions

Reply:
And maybe you should use the Oxford comma? I was under the impression we generally prefer it in the Prometheus universe (and yeah, we should probably have a formalized documentation style guide one day).